gigadb/app/tools/wasabi-migration/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apk update \
         ca-certificates \
         fuse \
         curl \
+        swatch \
```

Comment:
I would prefer that the Swatchdog Docker service be created by a simpler `Swatchdog-Dockerfile` which just contains the `swatch` tool installed and has a `CMD` attribute which returns the version of Swatchdog installed.